src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
I think I'd prefer a separate PR since I'm not sure if I will be able to get to it right away.